UnityProject/Assets/Scripts/Messages/Client/RequestHackingNodeConnections.cs

```diff
@@ -5,47 +5,44 @@
 
 namespace Messages.Client
 {
-	/// <summary>
-	///     Request hacking node data from the server.
-	/// </summary>
-	public class RequestHackingNodeConnections : ClientMessage<RequestHackingNodeConnections.NetMessage>
+	//Give me admin piz
```

Comment:
nani

Reply:
it's for script kiddie who uses control F on the code base and searches for Hack : P